source/sonata_tech.rst

```diff
@@ -576,3 +576,26 @@ Required fields for ``Spykfunc``:
 .. _specification: https://github.com/AllenInstitute/sonata/blob/master/docs/SONATA_DEVELOPER_GUIDE.md
 .. _enumeration: https://github.com/AllenInstitute/sonata/blob/master/docs/SONATA_DEVELOPER_GUIDE.md#nodes---enum-datatypes
 .. _Tsodyks Markram Model: https://www.pnas.org/content/94/2/719
+
+Format of the electrodes_file
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The format of the weights file is described below:
+
+.. table::
+
+  ============================================= ======================= ========== =================== ============= ==========================================
+  Group                                         Field                   Type       Shape               Requirement   Description
+  ============================================= ======================= ========== =================== ============= ==========================================
+  /electrodes/{electrodename}                   layer                   utf8       1                   Optional      Layer of the circuit in which {electrodename} is located. If the electrode is in a region without cortical layers, then "NA". If the electrode is outside the brain, then "Outside"
+  /electrodes/{electrodename}                   position                float32    3                   Mandatory     Position of {electrodename} in microns, in cartesian coordinates
+  /electrodes/{electrodename}                   region                  utf8       1                   Optional      Region in which {electrodename} is located
+  /electrodes/{electrodename}                   type                    utf8       1                   Optional      Either EEG or LFP
+  /electrodes/{electrodename}/{population_name} index                   uint64     1                   Mandatory     Index of the column corresponding to this electrode in /electrodes/{population_name}/scaling_factors
+  /electrodes/{population_name}                 scaling_factors         float64    Total_seg x N_elec  Mandatory     Scaling factor for each segment in the corresponding neuron, in mV/nA
```

Comment:
when describing `each segment`, is this the morphology segment (ie: between two sample points?), or the NEURON compartment?

If so, I'm not sure I follow why the section_ids field is needed, because (I think?) there is a 1-to-1 relationship between the segment and the section. Does that mean that `/{population_name}/section_ids` isn't needeed?

Reply:
My understanding is that most of the times there is a 1-to-1 match, but if not, only the sections in 'section_ids' will be reported. Again @joseph-tharayil correct me if I'm wrong

Reply:
Segment does refer to the NEURON compartment, not the morphology segment (so there are several segments per section, with each segment being in only one section). I don't think there should ever be a case where we exclude sections from the recording, so I do think that `/{population_name}/section_ids` is not needed

Reply:
removed